### ISSUE TITLE
[Bug] CLI doesn't work with big numbers

### ISSUE THREAD

**Issue by borispovod** [aptos-labs/aptos-core#8513]
# 🐛 Bug

CLI doesn't work with big numbers.

Aptos CLI 1.0.14

## To reproduce

```
aptos move run --function-id 0x310678fcefabd0e5a547ce26e32f49bda0ee47d6ddfa5b72b2dc424c14d17081::final_test::emit_vector_event \
  --args u256:"340282366920938463463374607431768211455" \
  --args u256:"340282366920938463463374607431768211455" \
  --args u256:"340282366920938463463374607431768211455" \
  --args u256:"340282366920938463463374607431768211455" \
  --args u256:"340282366920938463463374607431768211455"

OR 
aptos move run --function-id 0x310678fcefabd0e5a547ce26e32f49bda0ee47d6ddfa5b72b2dc424c14d17081::final_test::emit_vector_event \
  --args u256:"115792089237316195423570985008687907853269984665640564039457584007913129639935" \
  --args u256:"115792089237316195423570985008687907853269984665640564039457584007913129639935" \
  --args u256:"115792089237316195423570985008687907853269984665640564039457584007913129639935" \
  --args u256:"115792089237316195423570985008687907853269984665640564039457584007913129639935" \
  --args u256:"115792089237316195423570985008687907853269984665640564039457584007913129639935"
```

**Stack trace/error message**

```
error: Invalid value "u256:340282366920938463463374607431768211455" for '--args <ARGS>...': Unable to parse 'u256': error: a character is not in the range 0-9
```

**Comment by gregnazario** [comment]
Verified, looks like rust-clap is converting the large number to scientific notation:

```
./target/debug/aptos move run --function-id devnet::number::set_number --profile devnet --args u256:115792089237316195423570985008687907853269984665640564039457584007913129639935
ARG: 1.157920892373162e77
```

**Comment by 0xjinn** [comment]
Note: This is fixed with PR #9111 

Will be available in the next release

**Comment by 0xjinn** [comment]
The fix has been released. Closing this issue now
